Doc/c-api/exceptions.rst

```diff
@@ -422,6 +424,49 @@ Querying the error indicator
          }
 
 
+.. c:function:: PyObject *PyErr_Fetch1(void)
```

Comment:
I'd call them something like PyErr_GetRaisedException/PyErr_SetRaisedException.  We already have Get/SetHandledException.